src/interfaces/swig/shogun.i

```diff
@@ -239,4 +239,18 @@ namespace shogun
 %template(get_int_vector) CSGObject::get_vector_as_matrix_dispatcher<SGMatrix<int32_t>, int32_t>;
 #endif // SWIGJAVA
 
+%define PUT_ADD(sg_class)
+%template(put) CSGObject::put<sg_class, sg_class, void>;
```

Comment:
need to have this as put/add are now templates.
Also explains why I need 3 template parameters for put (there already is one with 2)